VisualizationBase/src/items/ViewItem.cpp

```diff
@@ -77,7 +78,7 @@ void ViewItem::insertColumn(int column)
 
 Model::Node* ViewItem::insertNode(Model::Node* node, int column, int row, int purpose)
 {
-	auto ref = ViewItemNode::withReference(node, purpose);
+	auto ref = ViewItemNode::withReference(node, purpose, this);
```

Comment:
This doesn't look right at all. Why does the node need to know about a `ViewItem` ? I guess I'll find out below, but I have a suspicion I wouldn't find it appropriate.